### ISSUE TITLE
List styles buttons should use the `aria-pressed` attribute

### ISSUE THREAD

**Issue by Mgsy** [ckeditor/ckeditor5#12463]
## 📝 Provide a description of the improvement

Currently, our buttons for the list styles don't use the `aria-pressed` attribute, depending on the button state. It should be added to improve the accessibility of this feature.

---

If you'd like to see this improvement implemented, add a 👍 reaction to this post.
